airflow/providers/google/cloud/hooks/vertex_ai/batch_prediction_job.py

```diff
@@ -358,3 +514,98 @@ def list_batch_prediction_jobs(
             metadata=metadata,
         )
         return result
+
+
+class BatchPredictionJobAsyncHook(GoogleBaseHook):
```

Comment:
I assume that the async hooks should inherit from `GoogleBaseAsyncHook`.

Reply:
You are totally right. I changed that and also updated ` get_job_service_client()` method in both async hooks in order to reuse `GoogleBaseAsyncHook.get_sync_hook()`. Thank you for noticing!